optuna/study/_dataframe.py

```diff
@@ -40,6 +40,8 @@ def _create_records_and_aggregate_column(
     column_agg: DefaultDict[str, Set] = collections.defaultdict(set)
     non_nested_attr = ""
 
+    metric_names = study._storage.get_study_system_attrs(study._study_id).get("metric_names")
```

Comment:
How about defining a constant in `study.py` instead of specifying `metric_names` in each module?

Some people might accidentally refer to `metric_name`, etc. It can be difficult to notice since `metric_names` is an optional key.